### ISSUE TITLE
Privacy issue: network connect to redirector.gvt1.com

### ISSUE THREAD

**Issue by benboeck** [heyman/heynote#128]
Hello,
thank you for the cool product, love it!

When trying it out I immediately got a warning that the software wants to connect to **redirector.gvt1.com** which essentially means Google gets a ping every time someone uses the app - bad for privacy.

This seems to be related to the download of spell checker files, as outlined in a similar issue in Signal-Desktop:
https://github.com/signalapp/Signal-Desktop/issues/5767

A possible solution is mentioned in the issue: https://github.com/signalapp/Signal-Desktop/commit/1c43e7501c30b959e2fb1b178135ab29b2b6af7a

Kind regards
Ben

**Comment by heyman** [comment]
Yes, this is the [default behaviour of Electron](https://www.electronjs.org/docs/latest/api/session#sessetspellcheckerdictionarydownloadurlurl):

> By default Electron will download hunspell dictionaries from the Chromium CDN. If you want to override this behavior you can use this API to point the dictionary downloader at your own hosted version of the hunspell dictionaries. We publish a hunspell_dictionaries.zip file with each release which contains the files you need to host here.

In theory, I agree, but I don't care enough to justify hosting custom dictionary files and then updating those files every time we upgrade Electron.
